index.d.ts

```diff
@@ -6,6 +6,13 @@ declare namespace camelcase {
 		@default false
 		*/
 		readonly pascalCase?: boolean;
+
+		/**
+		From `String.prototype.toLocaleUpperCase()`: The locale parameter indicates the locale to be used to convert to upper/lower case according to any locale-specific case mappings. If multiple locales are given in an Array, the best available locale is used. The default locale is the host environment’s current locale.
```

Comment:
> The default locale is the host environment’s current locale.

Is specified twice.

Reply:
I don't really see the point in mentioning `From `String.prototype.toLocaleUpperCase()`:`.

Reply:
Removed the reference and double default specification.